src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -339,6 +339,8 @@ The span finishes after the view appeared on the screen.
 
 ## Time to Full Display
 
+<Note>This feature is experimental and may have bugs.</Note>
```

Comment:
```suggestion
<Note>

This feature is experimental and may have bugs.

</Note>
```
There's been some rendering issues when the extra lines between the note tags aren't added, so let's just be safe here! This won't be an issue in the future when the docs move to next.js though!

Reply:
Thanks for the heads up!